### PR TITLE
fix: proper case if check

### PR DESCRIPTION
**Pull request by puneetloya**
## Description
its a bug fix for my previous pull request

### CODE REVIEW

File: python/ambassador/envoy/v2/v2listener.py [hunk 1]
```diff
@@ -811,8 +811,7 @@ def __init__(self, config: 'V2Config', service_port: int) -> None:
             if req_hdrs:
                 self.base_http_config["tracing"]["request_headers_for_tags"] = req_hdrs
 
-        proper_case = self.config.ir.ambassador_module.get('proper_case', False)
```
Comment (kflynn) [inline python/ambassador/envoy/v2/v2listener.py:814]: I actually think that this is better left alone -- what're you trying to solve here?

Comment (puneetloya) [reply python/ambassador/envoy/v2/v2listener.py:814]: the if check failed for me with the .get method, it was always evaluating as false(even after I set it to true in the config).

Comment (kflynn) [reply python/ambassador/envoy/v2/v2listener.py:814]: Gotcha, thanks.

**Comment by kflynn** [comment]
Superseded by #2555 since I needed to do hand testing -- thanks much!

**Pull request closed without merging**